test/v1-v2-compare-testPools.spec.ts

```diff
@@ -11,19 +11,19 @@ const testFiles = [
     'gusdBug',
     'gusdBugSinglePath',
     '25178485-blockKovan', // UI-506, WETH to WBTC Bug
-    // 'stable-and-weighted-token-btp-test',
-    // Added pools 'stable-and-weighted-token-btp-test' to element pools
-    //  and replaced the base token with DAI. Also
-    // added a weighted pool (last in json) with DAI and 0x000...001 to test 2 paths
-    'elementFinanceTest_multihop_1path',
-    'elementFinanceTest_multihop_2paths',
-    'elementFinanceTest_multihop_1path_swapExactOut',
-    'elementFinanceTest_multihop_2paths_swapExactOut', // Returning a second swap with -ve amount
-    'elementFinanceTestFourPools',
-    'elementFinanceTest1',
-    'elementFinanceTest2',
-    'elementFinanceTest3',
-    'elementFinanceTest4',
+    // // 'stable-and-weighted-token-btp-test',
+    // // Added pools 'stable-and-weighted-token-btp-test' to element pools
+    // //  and replaced the base token with DAI. Also
+    // // added a weighted pool (last in json) with DAI and 0x000...001 to test 2 paths
+    // 'elementFinanceTest_multihop_1path',
+    // 'elementFinanceTest_multihop_2paths',
+    // 'elementFinanceTest_multihop_1path_swapExactOut',
+    // 'elementFinanceTest_multihop_2paths_swapExactOut', // Returning a second swap with -ve amount
+    // 'elementFinanceTestFourPools',
+    // 'elementFinanceTest1',
+    // 'elementFinanceTest2',
+    // 'elementFinanceTest3',
+    // 'elementFinanceTest4',
     // 07/04/21 - Below have been checked and pass
```

Comment:
Should we add these tests back in? I noticed a couple cause `parseFixed` to throw a NUMERIC_FAULT error.

Reply:
Good catch, sorry I missed that I needed to reenable these.

There are quite a few lines which were already commented out. Can these other lines be removed now?

Reply:
I've addressed this now. It seems like in the test data USDC was given many more decimals places than it should have been.